docs/UserGuide.md

```diff
@@ -40,95 +44,94 @@ Format: `help`
+Edits a task to the scheduler specified by the index <br>
+Format: `edit INDEX NAME d/DATE` 
+
+Edits a floating task to the scheduler specified by the index <br>
+Format: `edit INDEX NAME` 
 
 Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+* `edit 1 Meeting st/20-03-12 13:00 et/20-03-12 14:00`
+* `edit 3 Do Homework1 d/19-02-12`
+* `edit 2 Do Homework2`
 
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
+#### Deleting an item: `delete`
+Delete the item selected by the user.<br>
 Format: `delete INDEX`
 
-> Deletes the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+> * The user needs to view Scheduler in a list to find index of item
 
 Examples: 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+* `delete 23`
 
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
+#### Finding all items containing any keyword in their name: `find`
+Finds items where their item names contain any of the given keywords.<br>
+Format: `find KEYWORD [MORE_KEYWORDS]`
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+> * The search is non-case sensitive. e.g `homework` will match `Homework`
+> * Only the name is searched.
+> * Only full words will be matched e.g. `CS` will not match `CS2103`
+> * Persons matching at least one keyword will be returned (i.e. `OR` search).
+    e.g. `Midterm` will match `Midterm Review`
 
 Examples: 
-* `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
-
-#### Clearing all entries : `clear`
-Clears all entries from the address book.<br>
-Format: `clear`  
+* `find Tutorial`<br>
+  Returns `CS2103 Tutorial` and words related to `tutorial`
+* `find CS Assignment Errand`<br>
+  Returns Any item having names `CS`, `Assignment`, or `Errand`
 
 #### Exiting the program : `exit`
 Exits the program.<br>
 Format: `exit`  
 
-#### Saving the data 
-Address book data are saved in the hard disk automatically after any command that changes the data.<br>
-There is no need to save manually.
+#### Saving the data: `path`
+Scheduler data are saved in the hard disk automatically after any command that changes the data. File path can be manually changed to a user-specified location.<br>
+Format: `path FILE_PATH`
 
```

Comment:
Saving the data and setting a new file path should be separated into different components.